pkg/kv/kvserver/rditer/testdata/TestReplicaDataIterator/r1/all/only/output

```diff
@@ -0,0 +1,9 @@
+echo
```

Comment:
What is "only" in this test name? Is this a leftover test after "only" was renamed to "user-only"? Or it's something else; but there is a "user-only" test too, so I'm not sure what the difference should be.

Reply:
You're right, this is an excess / unused file. Removed (along with others). Thanks!